### ISSUE TITLE
Automatic generating surface from a group of DICOM files

### ISSUE THREAD

**Issue by zerolong0309** [rii-mango/Papaya#164]
Thanks for the nice source code! It helped me a lot.
By the way, will there be any way to generate surface from a list of dcm files?

**Comment by martinezmj** [comment]
This has been asked lots of times.  Eventually it would be nice to add this, but it would be a pretty sizeable feature and so far there aren't any immediate plans to start working on it.

**Comment by niks2060** [comment]
Yes @martinezmj  this is a required feature which will complete this application. If possible we can start contributing at most we can to derive this feature.
Please let me know if I can help ?

**Comment by martinezmj** [comment]
The algorithm to convert volumetric image data to surface data is called "marching cubes".  Finding or putting together an implementation in JavaScript would be the first step.  Some links to start with:

- http://paulbourke.net/geometry/polygonise/
- https://stackoverflow.com/questions/9330334/a-good-example-code-of-how-to-implement-marching-cubes
- https://stackoverflow.com/questions/8550872/getting-started-with-marching-cubes-in-three-js
